src/database/entities/NotablePersonEvent.ts

```diff
@@ -24,6 +27,11 @@ const eventTypes: Record<NotablePersonEventType, string> = {
   appearance: '',
 };
 
+const reviewStatuses: Record<NotablePersonEventReviewStatus, string> = {
+  APPROVED: '',
+  DISAPPROVED: '',
```

Comment:
Should we use `ALLOWED`/`REMOVED` instead of `APPROVED`/`DISAPPROVED`?

I feel the words approved/disapproved communicate involvement with the content whereas allowed/removed are more neutral.